tests/ci/integration_tests_runner.py

```diff
@@ -464,6 +464,20 @@ def _get_parallel_tests_skip_list(repo_path):
             skip_list_tests = json.load(skip_list_file)
         return list(sorted(skip_list_tests))
 
+    @staticmethod
+    def _get_broken_tests_list(repo_path):
+        skip_list_file_path = f"{repo_path}/tests/integration/broken_tests.json"
```

Comment:
Why `broken_tests.json` but not some kind of `xfailed_test.json` ?

Reply:
Naming was chosen based on `run_impl()` understanding `BROKEN`, but not `XFAIL`. If another name for the file makes more sense, I don't have an issue with changing it. We could also add full support for `XFAIL` status if needed.

Reply:
Marking as resolved based on meeting last week